primap2/pm2io/_data_reading.py

```diff
@@ -1394,6 +1394,9 @@ def preferred_unit(entity: str, units: List[str], gwp_to_use: Optional[str]) ->
 
     In general, "Gg <substance> / year" will be preferred if it is compatible with the
     given input units. Otherwise, the first unit from units will be preferred.
+    If gwp_to_use is not None native units without GWP will be preferred. If it is
+    not set but conversion without gwp context is impossible Gg <unit_substance> / yr
+    will be returend as preferred unit if the first unit from units can be converted.
```

Comment:
Isn't `Gg <unit_substance> / yr` a native unit? If so, then I don't understand the docs. As far as I understand: If `gwp_to_use` is `None` ("not set"), then native units should be preferred, and then if a GWP context is not available, things are weird.

Reply:
Ah, maybe the only info missing there is this:
> If gwp_to_use is None, and in `units` are GWP units as well as native units and conversion between them without gwp context is impossible, `Gg <unit_substance> / yr` will be returned as preferred unit if at least the first unit from `units` can be converted.

That way, I understand *what* it is doing (but not why).